iOS Example Frame/iOS Example FrameUITests/XCUIApplication+Extension.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  XCUIApplication+Extension.swift
+//  iOS Example FrameUITests
+//
+//  Created by Alex Ioja-Yang on 24/08/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import XCTest
```

Comment:
Where on the PR is the UI test ? What are we testing on this PR token creation ? or validation errors ?

Reply:
Interesting, the file is on my disk at this level. I'm confused how its not here too, been battling Bitrise today 😮‍💨. Will update once I understand how its not here 🤕